src/NovelRT.Interop/Audio/NrtAudioService.cpp

```diff
@@ -138,7 +221,16 @@ NrtResult Nrt_AudioService_loadSound(NrtAudioService service, char* input, unsig
     }
 
     auto serv = reinterpret_cast<Audio::AudioService*>(service);
-    *output = serv->loadSound(input);
+
+    try
+    {
+      *output = serv->loadSound(input);
+    }
+    catch (const Exceptions::NotInitialisedException)
+    { // todo: handle error message
```

Comment:
This looks like a custom exception. Do we have an internal function for the error message?